src/mymonero/MyMoneroApi.js

```diff
@@ -2,7 +2,7 @@
 
 import type { MyMoneroCoreBridge } from 'react-native-mymonero-core'
 
-const nettypeUtils = require('./cryptonote_utils/nettype.js')
+const nettypeUtils = require('@mymonero/mymonero-nettype')
```

Comment:
Same question as before but for the other libs.

Reply:
Same answer for the others. The libs we have copied all have `async` added to them.